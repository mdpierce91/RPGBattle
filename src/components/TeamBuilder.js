import React from 'react';
import UnitList from './UnitList';

const TeamBuilder = () => {
    return (
        <div>
            
            <UnitList />
            
        </div>
    )
}

export { TeamBuilder as default };