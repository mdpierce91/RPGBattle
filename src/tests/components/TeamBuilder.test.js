import React from 'react';
import { shallow } from 'enzyme';
import TeamBuilder from '../../components/TeamBuilder';

let wrapper;

beforeEach(() => {
    wrapper = shallow(<TeamBuilder/>);
});

test('should render TeamBuilder', () => {
    expect(wrapper).toMatchSnapshot();
});