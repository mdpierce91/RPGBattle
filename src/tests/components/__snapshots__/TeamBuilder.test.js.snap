// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TeamBuilder 1`] = `
<div>
  <Connect(UnitList) />
</div>
`;
